go.mod

```diff
@@ -3,145 +3,148 @@ module github.com/scylladb/scylla-operator
 go 1.18
 
 require (
-	cloud.google.com/go v0.81.0
-	github.com/aws/aws-sdk-go-v2/config v1.9.0
-	github.com/aws/aws-sdk-go-v2/feature/ec2/imds v1.7.0
+	cloud.google.com/go/compute v1.9.0
+	github.com/aws/aws-sdk-go-v2/config v1.17.2
+	github.com/aws/aws-sdk-go-v2/feature/ec2/imds v1.12.13
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/c9s/goprocinfo v0.0.0-20210130143923-c95fcf8c64a8
 	github.com/cenkalti/backoff v2.2.1+incompatible
-	github.com/evanphx/json-patch v4.11.0+incompatible
+	github.com/evanphx/json-patch v4.12.0+incompatible
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-git/go-git/v5 v5.4.2
-	github.com/go-openapi/errors v0.20.1
-	github.com/go-openapi/runtime v0.20.0
-	github.com/go-openapi/strfmt v0.20.3
-	github.com/go-openapi/swag v0.19.15
-	github.com/go-openapi/validate v0.20.3
-	github.com/gocql/gocql v0.0.0-20211015133455-b225f9b53fa1
-	github.com/google/go-cmp v0.5.6
+	github.com/go-openapi/errors v0.20.3
+	github.com/go-openapi/runtime v0.24.1
+	github.com/go-openapi/strfmt v0.21.3
+	github.com/go-openapi/swag v0.22.3
+	github.com/go-openapi/validate v0.22.0
+	github.com/gocql/gocql v1.2.0
```

Comment:
@zimnx is this one ok to bump?

Reply:
this one shouldn't matter becasue we need to have a replace directive in go.mod. You can update our fork version though to latest one.

Reply:
right, updated